grinder/top_ports.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
```

Comment:
We don't have underscores in grinder modules. So, maybe, you should call it something like `topports.py` or `masscantopports.py` or `topmasscanports.py`